### PR TITLE
#2559. Add more augmented expression tests for class-like declarations

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Augmentation-libraries/augmented_expression_A10_t26.dart [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Consider a non-augmenting member declaration Dm that occurs
+/// inside an augmenting type declaration Dt. A compile-time error occurs if the
+/// identifier `augmented` occurs in Dm.
+///
+/// @description Checks that it is a compile-time error if an augmenting class
+/// declaration extends a type named `augmented`.
```
Comment (eernstg) [inline LanguageFeatures/Augmentation-libraries/augmented_expression_A10_t26.dart:10]: Perhaps:
```suggestion
/// declaration extends a type that contains the name `augmented`.
```

Similarly for a few more tests below.

**Review by eernstg — COMMENTED** [review]
Looks good! A couple of words in the description do not seem to match the actual test.

**Comment by sgrekhov** [comment]
Updated. PTAL.

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**